scripts/build-docs

```diff
@@ -3,35 +3,35 @@
 EXAMPLE_WORKSPACE='Example/Backpack.xcworkspace'
 
 function documentUIKit() {
-  sourcekitten doc -- -workspace $EXAMPLE_WORKSPACE -scheme "Backpack Native" >swiftDoc.json
+  sourcekitten doc -- -workspace $EXAMPLE_WORKSPACE -scheme Backpack >swiftDoc.json
 
   $(dirname "$0")/create-docs-header
 
   sourcekitten doc --objc $(pwd)/Backpack/BackpackDocHeaders.h -- -x objective-c -isysroot $(xcrun --show-sdk-path --sdk iphonesimulator) -I $(pwd) -fmodules >objcDocs.json
-  jazzy --sourcekitten-sourcefile objcDocs.json,swiftDoc.json -o $outputDir/uikit
+  jazzy --sourcekitten-sourcefile objcDocs.json,swiftDoc.json -o "$outputDir/versions/$version/uikit"
 
   rm Backpack/BackpackDocHeaders.h
   rm objcDocs.json
   rm swiftDoc.json
 }
 
 function documentSwiftUI() {
-  sourcekitten doc -- -workspace $EXAMPLE_WORKSPACE -scheme "Backpack SwiftUI" >swiftDoc-swiftui.json
-  jazzy --sourcekitten-sourcefile swiftDoc-swiftui.json -o $outputDir/swiftui
+  sourcekitten doc -- -workspace $EXAMPLE_WORKSPACE -scheme "Backpack-SwiftUI" >swiftDoc-swiftui.json
+  jazzy --sourcekitten-sourcefile swiftDoc-swiftui.json -o "$outputDir/versions/$version/swiftui"
   rm swiftDoc-swiftui.json
 }
 
 function buildRedirectHTML() {
   echo '<!DOCTYPE html>
   <html>
-    <meta http-equiv="refresh" content="0;url=/ios/versions/latest" />
+    <meta http-equiv="refresh" content="0;url=/ios/versions/latest/index.html" />
```

Comment:
Will this redirect to swiftui then? Or does this need an extra swiftui or uikit in the url?

Reply:
This will redirect to the latest/index.html, which all it does itself is redirect to swiftui as default. I guess we want that to have both a 'latest' url you can go to, to always use the latest version, but also we're using this latest page for when we go to a non existant url